zulipterminal/api_types.py

```diff
@@ -1,6 +1,5 @@
 """
-Preliminary Zulip API types defined in python,
-to allow type checking
+Preliminary Zulip API types defined in python, to allow type checking
```

Comment:
I'm not sure why the change in this 3rd (?) commit isn't in the original commit?

Reply:
The original commit had line breaks for longer docstrings. After your comments, I removed those line breaks, which were added in the third commit. As said above, I'll make the commit flow cleaner.